scripts/benchmarks/build_and_run.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+#
+# This script is meant to be used in a new machine, to build LLVM, TPP-MLIR
+# and run all benchmarks. This should work in the same way as our local tests
+# and reproduce our numbers on local/cloud machines.
+
+# Include common utils
+SCRIPT_DIR=$(realpath $(dirname $0)/..)
+source ${SCRIPT_DIR}/ci/common.sh
+
+# Install packages needed
+if [ "$(is_linux_distro Ubuntu)" == "YES" ]; then
+  sudo apt update && \
```

Comment:
I suggest using conda, this is much more portable.

Reply:
Conda faffles with `.bashrc` and needs reloading stuff. I found the `dnf` install line for "Amazon Linux" that just worked on different instances on AWS.